fast_llm/data/preparator/hf_processors/configs/base.py

```diff
@@ -0,0 +1,19 @@
+import abc
+import typing
+import datasets
+
+from fast_llm.config import TypeableConfig, config_class
+from fast_llm.utils import Registry
+
+
+class Applicable:
```

Comment:
Not following the point of this. Consider making an abstract base class for processors instead.